functions/go/starlark/starlark/config.go

```diff
@@ -34,6 +34,8 @@ type StarlarkRun struct {
 
 func (sr *StarlarkRun) Config(fnCfg *fn.KubeObject) error {
 	switch {
+	case fnCfg.IsEmpty():
+		return fmt.Errorf("FunctionConfig is missing. Expect `ConfigMap` or `StarlarkRun`")
```

Comment:
@mengqiy  This is actually not needed for the new SDK. if not check, this will fall into the default case
`functionConfig` must be either ...`

Reply:
Or we can always check the IsEmpty. wdyt?

Reply:
It seems this also applies to the other 2 functions, right?

Reply:
If not check, the falls into the default with error message 
"functionConfig` must be either ConfigMap or StarlarkRun, but we got: " . which gives users the same kind of suggestion as  `IsEmpty`

Reply:
Basically the SDK fixes the nil pointer references when calling a KRM methods. This `IsEmpty` is merely for better user messaging. Not to handle risk conditions.